components/google_classroom/README.md

```diff
@@ -15,3 +15,5 @@ API:
   assignments for their classes
 - A student submission system that allows teachers to view and grade student
   submissions
+
+Pipedream's use and transfer of information received from Google APIs to any other app will adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```

Comment:
```suggestion
Pipedream's use and transfer of information received from Google APIs to any other app adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```

Reply:
Changed "will adhere" to "adhere" to follow Google's Developer docs style guide (use present tense).